src/components/organisms/about/about.scss

```
@@ -0,0 +1,34 @@
+@use "../../variables" as *;
+
+.container {
+  padding: 0px;
+  margin: 100px 0px;
```

```suggestion
  padding-top: 100px;
  padding-bottom: 100px;
```
`cards` と同様にmarginは `.container` のものを使います。
上下に余白を付けたいので、`padding` を設定しましょう。
`padding: 0 100px` としないのは、 `.container` にもとからついている横方向の `padding` を上書きしないためです。